numpy/_globals.py

```diff
@@ -89,3 +92,19 @@ def __repr__(self):
 
 
 _NoValue = _NoValueType()
+
+class CopyMode(enum.Enum):
```

Comment:
Since all enumeration values are integer anyway, would it make sense to subclass `enum.IntEnum` instead?

Reply:
Definitely. Thanks for the suggestion.